pkg/exchange/paperwallet.go

```diff
@@ -109,6 +123,8 @@ func (p *PaperWallet) Summary() {
 	fmt.Println("FINAL PORTFOLIO = ", total+baseCoinValue, p.baseCoin)
 	fmt.Printf("GROSS PROFIT    =  %f %s (%.2f%%)\n", profit, p.baseCoin, profit/p.initialValue*100)
 	fmt.Printf("MARKET CHANGE   =  %.2f%%\n", avgMarketChange*100)
+	fmt.Printf("VOLUME          =  %.2f %s\n", volume, p.baseCoin)
+	fmt.Printf("COSTS (0.001*V) =  %.2f %s\n", volume*0.001, p.baseCoin)
```

Comment:
Why is it multiplied by 0.001?

Reply:
https://www.binance.us/en/fee/schedule    binance is really cheap compared to other brokers.  the 0.001 Is just to give a a simple indication of the costs of all transactions.  As you can see in the demo. My Strategy gives a 140% return. but the VOLUME is insane and the costs of moving that volume is bigger then  the profit.

Reply:
added an (estimation) comment to the string. 0.001 is th e 0.1% costs of volume.